internal/research/opentelemetry/deployment-metrics.md

```diff
@@ -0,0 +1,59 @@
+# OpenTelemetry Collector Usage for Metrics
+
+OpenTelemetry is a project to handle telemetry data (logs, metrics and traces) in a vendor-neutral way. It provides SDKs to instrument applications, APIs and processing tools (collector).
```

Comment:
```suggestion
OpenTelemetry is a project to handle telemetry data (logs, metrics and traces) in a vendor-neutral way. It provides SDKs to instrument applications, APIs and processing tools ([Collector](https://opentelemetry.io/docs/collector/)).
```

When I tried to find out whether Collector must be capitalized, I found this link. I believe it doesn't hurt to add it. Also, they seem to capitalize it all the time.

Reply:
There's only one `OpenTelemetry Collector`. Would make sense to capitalize that.